sdks/go/pkg/beam/core/runtime/exec/sdf.go

```diff
@@ -217,6 +218,11 @@ type ProcessSizedElementsAndRestrictions struct {
 	PDo *ParDo
 
 	inv *ctInvoker
+
+	// rt allows this unit to send out restriction trackers being processed.
```

Comment:
Typo" Rt
```suggestion
	// Rt allows this unit to send out restriction trackers being processed.
```

Reply:
Fixed. (Also switched the channel from `*RTracker` to `RTracker`)

Reply:
Ah yes. Good catch. Don't use pointers to interface types :)